src/main/java/sirius/pasta/tagliatelle/emitter/ConstantEmitter.java

```diff
@@ -62,6 +62,33 @@ public ConstantEmitter append(String stringToAppend) {
         return this;
     }
 
+    public ConstantEmitter stripLeading() {
+        if (Strings.isEmpty(value)) {
+            return this;
+        }
+
+        value = value.stripLeading();
+        return this;
+    }
+
+    public ConstantEmitter stripTrailing() {
+        if (Strings.isEmpty(value)) {
+            return this;
+        }
+
+        value = value.stripTrailing();
+        return this;
+    }
```

Comment:
We could instead add this methods to `Emitter` directly as NOOP, which would avoid the `instanceof` checking.
Would probably be a bit cleaner, but on the expense of more boilerplate code; you decide.

Reply:
Yeah, I am just not sure whether the respective methods would have semantic significance within `Emitter`. Let's see, I am still thinking whether we really should strip all the whitespace, or whether it is enough (and safer) to only omit the last line break.

Reply:
reworked